contracts/bridge/libraries/BridgeStructs.sol

```diff
@@ -22,8 +23,13 @@ struct SynapseParams {
     uint8 tokenIndexTo;
 }
 
+struct PoolToken {
+    bool isWeth;
```

Comment:
Do we want to use the naming Weth / HandleETH or WGAS / HandleGas? No strong opinion.

Reply:
I think having ETH_ADDRESS / Weth / HandleETH in the contract code is fine. Pretty self-explanatory, especially given that the contract is usually reviewed as standalone, so the multi-chain context is kind of omitted.

In the multichain deployments / deploy scripts / etc I think using WGAS notation will be preferable